04.ls/ls.rb

```diff
@@ -33,14 +34,14 @@ def main
 
 def display_file_names_with_long(path, file_names)
   file_paths = file_names.map { |file_name| path + file_name }
-  total_block_size = get_total_block_size(file_paths)
+  total_block_size = file_paths.sum { |file_path| File.stat(file_path).blocks }
   column_name_to_width = get_column_name_to_width(file_paths)
   puts "total #{total_block_size / 2}"
```

Comment:
```suggestion
  puts "total #{total_block_size / 2}"
  column_name_to_width = get_column_name_to_width(file_paths)
```

変数の寿命を短くした方がいいかも
https://bootcamp.fjord.jp/pages/311